Part-1-Functions

```diff
@@ -67,6 +70,21 @@ def stringBits(str):
 
 
 def end_other(a, b):
+str1="Hiabc"
+str2="abc"
+
+def endwith(str1,str2):
+    str1=str1.lower()
+    str2=str2.lower()
+    l=str2
+    if str1.endswith(l):
+        print("True")
+    else:
+        print("False")
+
+
+endwith(str1,str2)
+  
```

Comment:
This checks only if the the string 1 ends with string 2, What if the str 1 is abc and string 2 is abXabc?